src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface ReportService {
+    String createReport(Map<String, Integer> groupedTransactions);

```

Comment:
```suggestion
    String createReport();
```
Do not pass Map in this method. Go to Storage inside the implementation.